app/views/deductionBreakdown.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+ @import models.calculation.CalcDisplayModel
+ @import views.html.templates.main_template
+ @import implicits.ImplicitCurrencyFormatter._
+ @import views.html.helpers.backLinkHelper
+
+@(calcModel: CalcDisplayModel, taxYear: Int)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig, user: auth.MtdItUser[_])
+
+@tableRow(messageCode: String, dataItem: Option[BigDecimal]) = {
+  @if(dataItem.isDefined) {
+   <tr id="@messageCode">
+    <td>@messages(s"deduction_breakdown.table.$messageCode")</td>
+    <td class="numeric">@{dataItem.get.toCurrency}</td>
+   </tr>
+  }
```

Comment:
can be simplified, doesn't need the .isDefined and .get, can use dataItem.map { data => data is not optional here anymore }